exosphere-shared/templates/add-service/exoservice-es6-mongodb/features/support/env.js

```diff
@@ -29,7 +29,6 @@ module.exports = function() {
   this.After(function() {
     this.exocom && this.exocom.close()
     this.process && this.process.close()
-    this.process && this.process.closePort()
```

Comment:
This change is not related to WebPack. Extract into another PR with explanation what it achieves?